primitives/src/v2/mod.rs

```diff
@@ -28,7 +28,8 @@ use parity_util_mem::MallocSizeOf;
 
 /// Information about validator sets of a session.
 #[derive(Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
-#[cfg_attr(feature = "std", derive(PartialEq, MallocSizeOf))]
+#[cfg_attr(feature = "std", derive(PartialEq, MallocSizeOf, Default))]
+#[cfg_attr(all(feature = "std", test), derive(Default))]
```

Comment:
No default for test, add an explicit `dummy_session_info` to the `primitives-test` crate

Reply:
Forgot to fix this one before pushing ...